src/applications/personalization/view-dependents/layouts/ViewDependentsLists.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import ViewDependentsList from '../components/ViewDependentsList/ViewDependentsList';
+
+class ViewDependentsLists extends Component {
+  render() {
+    const onAwardDependents = [
+      {
+        name: 'Billy Bush',
+        onAward: true,
+        social: '312-314-1415',
+        birthdate: '05/05/1982',
+        age: 32,
+      },
+    ];
+
+    const notOnAwardDependents = [
+      {
+        name: 'De Angelo Barksdale',
+        spouse: true,
+        onAward: true,
+        social: '312-314-1415',
+        birthdate: '05/05/1982',
+        age: 32,
+      },
+    ];
+
+    const onAwardSubhead = (
+      <span>
+        Dependents on award have been added to you disability claim.{' '}
+        <strong>
+          If a dependents status has changed, you need to let the VA know.
+        </strong>
+      </span>
+    );
+
+    const notOnAwardSubhead = (
+      <span>
+        Dependents not on award may be awaiting a decision, or they were once
+        added to your disability claim and their status has changed.{' '}
+        <strong>Let the VA know if a dependent's status has changed</strong> and
+        they are now eligible to be added to your disability claim.
+      </span>
+    );
+
+    return (
+      <span>
```

Comment:
This could be returned as a fragment to avoid nesting the child components in a `<span>`, or maybe a `<div>`  to make it a little more semantic.

Reply:
Yep, updated to a fragment.